src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -289,8 +292,28 @@ export const ControlGroupStrings = {
       }),
     getRemoveButtonTitle: () =>
       i18n.translate('controls.controlGroup.floatingActions.removeTitle', {
-        defaultMessage: 'Remove control',
+        defaultMessage: 'Delete control',
       }),
+    clearAction: {
+      [OPTIONS_LIST_CONTROL]: {
+        getClearButtonTitle: () =>
+          i18n.translate('controls.controlGroup.floatingActions.optionsList.clearTitle', {
+            defaultMessage: 'Clear selections',
+          }),
+      },
+      [RANGE_SLIDER_CONTROL]: {
+        getClearButtonTitle: () =>
+          i18n.translate('controls.controlGroup.floatingActions.rangeSlider.clearTitle', {
+            defaultMessage: 'Clear range selection',
+          }),
+      },
+      [TIME_SLIDER_CONTROL]: {
+        getClearButtonTitle: () =>
+          i18n.translate('controls.controlGroup.floatingActions.timeSlider.clearTitle', {
+            defaultMessage: 'Clear time selection',
+          }),
+      },
```

Comment:
Not sure if we want the copy to be unique for each control type...

Reply:
I would just have it say `Clear`.

Reply:
I would go the route of simplifying and removing the word control from all the tooltips.

Reply:
I like it :+1: Thanks!

![Jun-14-2023 11-31-04](https://github.com/elastic/kibana/assets/8698078/fe94ff7f-c678-44a8-8503-720800c7db90)